Firmware/klipper_configurations/SKR_1.4/Voron_1_SKR_14_Config.cfg

```diff
@@ -402,12 +402,25 @@ gcode:
 [z_tilt]
 ##	Use Z_TILT_ADJUST to level the bed left to right.
 ##	z_positions: Location of toolhead
-z_positions:
+
+##--------------------------------------------------------------------
+## Uncomment below for 250mm build
+#z_positions:
 	#-35, 180
 	#290, 180
-points:
+#points:
 	#30, 155
 	#220, 155
+
+## Uncomment below for 300mm build
+#z_positions:
+	#-35, 180
```

Comment:
These should shift back slightly since the bed is deeper, so we should add 25mm to the Y distance. 

-35, 205
340, 205

Reply:
@jdlongenecker should the Y distance for `points` be different between a 250 and 300 too?  Both are at 155 right now. Should a 250 be using 130?